Documentation/gettingstarted/kubeproxy-free.rst

```diff
@@ -254,6 +254,7 @@ This setting can be changed through the ``global.nodePort.mode`` helm option to
 ``dsr`` in order to let Cilium's BPF NodePort implementation operate in DSR mode.
 In this mode, the backends reply directly to the external client without taking
 the extra hop, meaning, backends reply by using the service IP/port as a source.
+DSR currently requires Cilium to be deployed in :ref:`arch_direct_routing`.
```

Comment:
Maybe add ", i.e. it won't work in the tunneling mode" to add more emphasis on this?